test/Client.test.ts

```diff
@@ -176,6 +176,37 @@ describe('Client', () => {
         convos.forEach((m, i) => assert.equal(m.content, messages[i]))
       })
 
+      it('query historic messages', async () => {
+        const c1 = await testCase.newClient()
+        const c2 = await testCase.newClient()
+        await c1.getUserContactFromNetwork(c2.address)
```

Comment:
GetUserContactFromNetwork bypasses the contact bundle cache. Is that the intended behaviour here?

Reply:
Yea I had to put this here to wait for the contact to be available on the network, otherwise it would race with the publish and fail.